submodules/wizard/wizard.scss

```diff
@@ -373,6 +373,14 @@ $color-highlight: #dbdbde;
 			letter-spacing: 0px;
 		}
 
+		.field-label-info {
+			@extend .field-label-horizontal;
+			font-style: italic;
+			font-size: 12px;
```

Comment:
I think it looks a little bit off with a smaller font than the rest of the labels, but I leave this choice to you, because this scenario was not taken into account in the mockups :slightly_smiling_face:

About the CSS, can you remove the two extra empty lines? To keep the file contents tidy and even.